app/assets/javascripts/discourse/widgets/notification-item.js.es6

```diff
@@ -10,13 +10,14 @@ import {
   formatUsername
 } from "discourse/lib/utilities";
 import { setTransientHeader } from "discourse/lib/ajax";
-import { userPath } from "discourse/lib/url";
+import { groupPath, userPath } from "discourse/lib/url";
 import { iconNode } from "discourse-common/lib/icon-library";
 
 const LIKED_TYPE = 5;
 const INVITED_TYPE = 8;
 const GROUP_SUMMARY_TYPE = 16;
 export const LIKED_CONSOLIDATED_TYPE = 19;
+const GROUP_INVITE = 21;
```

Comment:
@jjaffeux showed me a really good pattern to use that will allow us to not have to hardcode server side values on the client side. Can you try to adopt this pattern https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/components/concerns/category-search-priorities.js.es6.erb? 

Note that adding the `.erb` extension prevents the editor from identifying the file as `js` or `es6` so I usually put the `erb` files under `concerns`.